en/docs/design/api-security/oauth2/grant-types/kerberos-oauth2-grant.md

```diff
@@ -1,4 +1,4 @@
-# Kerberos OAuth2 Grant
+following information is added to you# Kerberos OAuth2 Grant
```

Comment:
Need to remove this change to topic

Reply:
Fixed with [f223bdb](https://github.com/wso2/docs-apim/pull/7315/commits/f223bdb4f8ba89fcb20bf9aa6a367bb2e80b600b)